java/streaming/src/test/java/com/ray/streaming/demo/WordCount.java

```diff
@@ -0,0 +1,72 @@
+package com.ray.streaming.demo;
+
+import com.ray.streaming.api.context.RayContext;
+import com.ray.streaming.api.function.impl.FlatMapFunction;
+import com.ray.streaming.api.function.impl.KeyFunction;
+import com.ray.streaming.api.function.impl.ReduceFunction;
+import com.ray.streaming.api.function.impl.SinkFunction;
+import com.ray.streaming.api.stream.StreamSource;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class WordCount {
+
```

Comment:
Can you add some simple tests for the plan construction as well as for individual operators (e.g. join)?

Reply:
ok~